python/doc/theory/probabilistic_modeling/conditional_distributions.rst

```diff
@@ -0,0 +1,402 @@
+  parameters set of each of its atoms (the weights of the mixture are not considered as parameters).
+- each component :math:`Y_i` is continuous or discrete: e.g., it can not be a :class:`~openturns.Mixture` of
+  discrete and continuous distributions, (so that the random vector :math:`\vect{Y}` may have some discrete components and some continuous components),
+- the copula of :math:`\vect{Y}` is continuous: e.g., it can not be the :class:`~openturns.MinCopula`.
+
+We define:
+
+.. math::
+
+    p_{\vect{Y}}(\vect{y}) = \left( \prod_{i=1}^\inputDim p_i(y_i) \right) c(F_1(x_1), \dots, F_d(x_d))
+
+where:
+
+- :math:`c` is the probability density copula of :math:`\vect{Y}`,
+- if :math:`Y_i` is a continuous component, :math:`p_i` is its probability density function,
+- if :math:`Y_i` is a discrete component, :math:`p_i = \sum_{y^i_k \in \cS^i} \Prob{Y_i = y^i_k} \delta_{y^i_k}` where
+  :math:`\cS^i = \{ y^i_k \}` is its support and :math:`\delta_{y^i_k}` the Dirac distribution centered on
+  :math:`y^i_k`.
+
+
+Then, the PDF of :math:`\inputRV` is defined by:
+
+.. math::
+
+    p_{\vect{X}}(\vect{x}) = \int p_{\vect{X}|\vect{\Theta}=g(\vect{y})}(\vect{x}|g(\vect{y})) p_{\vect{Y}}(\vect{y})\di{\vect{y}}
+
+with the same convention as for :math:`\vect{Y}`.
+
+
+Case 3: Condition a joint distribution by some values of its margins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let :math:`\inputRV` be a random vector of dimension :math:`\inputDim`. Let :math:`\cI \subset \{1, \dots, \inputDim \}`
+be a set of indices of components of  :math:`\inputRV`, :math:`\overline{\cI}` its complementary in
+:math:`\{1, \dots, \inputDim \}` and :math:`\vect{x}_\cI` a real vector of dimension equal to the cardinal of :math:`\cI`.
+The objective is to create the distribution of:
+
+.. math::
+
+    \inputRV_{\overline{\cI}}|\inputRV_\cI = \vect{x}_\cI
+
+
+See the class :class:`~openturns.experimental.DeconditionedDistribution`.
+
+This class requires the following features:
+
+- each component :math:`X_i` is continuous or discrete: e.g., it can not be a :class:`~openturns.Mixture` of
+  discrete and continuous distributions,
+- the copula of :math:`\inputRV` is continuous: e.g., it can not be the :class:`~openturns.MinCopula`,
+- the random vector :math:`\inputRV_{\overline{\cI}}` is continuous or discrete: all its components must be discrete
+  or all its components must be continuous,
+- the random vector :math:`\inputRV_{\cI}` may have some discrete components and some continuous components.
+
+Then, the pdf (probability density function if :math:`\inputRV_{\overline{\cI}}` is continuous or probability distribution function if
+:math:`\inputRV_{\overline{\cI}}` is discrete) of :math:`\inputRV_{\overline{\cI}}|\inputRV_\cI = \vect{x}_\cI` is defined by (in the following
+expression, we assumed a particular order of the conditioned components among the whole set of components for easy reading):
+
+.. math::
+    :label: condPDF
```

Comment:
it seems you're not allowed to use the same labels as in the class docstring

Reply:
ok